internal/proxy/makeproxy.sh

```diff
@@ -38,4 +38,7 @@ repo_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." >/dev/null 2>&1 && pwd
 for path in "." "internal/contributebot" "samples/appengine"; do
   cd "$repo_root/$path"
   go mod download
+  # We need to install goveralls because Travis uses it for tests and will look
+  # for it in the module proxy.
+  go install github.com/mattn/goveralls
```

Comment:
Nit: I think `go get` is sufficient here.

Reply:
Done.

Tested it by reruning the script with a fresh dir in `/tmp` and observing that goveralls was downloaded (I didn't want to disturb the proxy bucket itself, there be dragons...)